README.md

```diff
@@ -77,3 +77,45 @@ Create your message in $message and
 ```
 $mailer->sendMessage($message);
 ```
+
+## Retrieving receivers from objects
+
+It is possible to use a Determiner to retrieve the receivers (email/name) from an email configuration.
+For instance from an object:
+
+``` php
+$profile = new Profile();
+$profile->setEmail('foobarz@example.com');
+$profile->setFullname('Foo Barz');
+
+$emailConfiguration = ['email' => '{{email}}', 'name' => '{{full_name'}}];
+$determiner = new RonRademaker\Mailer\Receiver\Determiner();
+$receiver = $determiner->getReceivers($profile, null, $emailConfiguration);
+// results in ['email' => 'foobarz@example.com', 'name' => 'Foo Barz'];
+```
+Or an object with chained field configuration:
+
+``` php
+$profile = new Profile();
+$profile->setEmail('foobarz@example.com');
+$profile->setFullname('Foo Barz');
+$company = new Company();
+$company->setContactperson($profile);
+
+$emailConfiguration = ['email' => '{{profile.email}}', 'name' => '{{profile.full_name'}}];
```

Comment:
It'd make more sense to me if this would be ```contactperson.email``` and ```contactperson.full_name```, which would resolve to:

```php
$company->getContactperson()->getEmail();
$company->getContactperson()->getFullName();
```